### PR TITLE
groups/sig-testing: add k8s-infra-cherrypick-robot

### PR DESCRIPTION
**Pull request by spiffxp**
Related:
- Part of: https://github.com/kubernetes/test-infra/issues/23469

### CODE REVIEW

**Comment by alvaroaleman** [comment]
:tada: yay!
/lgtm
/hold

Feel free to cancel the hold if you want it to merge  :)

File: groups/sig-testing/groups.yaml [hunk 1]
```diff
@@ -182,7 +182,19 @@ groups:
...
       ReconcileMembers: "true"
       WhoCanPostMessage: "ANYONE_CAN_POST"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+    managers:
+    - ameukam@gmail.com
+    - spiffxp@gmail.com
+    - spiffxp@google.com
+
+  - email-id: k8s-infra-cherrypick-robot@kubernetes.io
```
Comment (ameukam) [inline groups/sig-testing/groups.yaml:191]: @spiffxp Will be great to see the password for this account in a 1password vault.

**Pull request merged**